### ISSUE TITLE
Configuration save

### ISSUE THREAD

**Issue by lincalinca** [urandom/gearshift#49]
I've just switched devices and have had to set up my profiles all over again. Is there a way that the config can be saved down to upload again (Transdrone does this)? Or is it there already and I've just missed it? Better yet, saving config to the cloud would be better, in the event that my phone or tablet is stolen or lost, I can recover quickly without having to access the original config file.

tl;dr:
1. Can we save configuration, with all server access, IP/hostname, folder configuration and everything?
2. Happy to start with this as a separate file to transfer, but would prefer to save to cloud, or configure a Gear Shift account to host all of my config files (limited to, say, 3, and to pay for more than that).

**Comment by urandom** [comment]
With android 6.0, all configuration is backed up in your google drive via the official backup method. I've been told by multiple people that it works. There are no intentions to set up anything else than the official backup method.

**Comment by lincalinca** [comment]
Understood. Wasn't aware, but I'll bear that in mind in 12 months when I get a new handset. Cheers.
